youtube_dl/downloader/common.py

```diff
@@ -358,7 +358,14 @@ def download(self, filename, info_dict):
                     else '%.2f' % sleep_interval))
             time.sleep(sleep_interval)
 
-        return self.real_download(filename, info_dict)
+        real_download_result = self.real_download(filename, info_dict)
+
+        if real_download_result and self.params.get('test', False):
+            f = open(filename, 'ab')
+            f.truncate(self._TEST_FILE_SIZE)
```

Comment:
Indeed, I believe the _Allow entire file to match or first 10241 bytes_ may be more suitable?